app/code/BelodubrovskyiAn/Lesson10Module/Ui/DataProvider/Product/Form/Modifier/NewField.php

```diff
@@ -0,0 +1,77 @@
+<?php
+namespace BelodubrovskyiAn\Lesson10Module\Ui\DataProvider\Product\Form\Modifier;
+
+use Magento\Catalog\Model\Locator\LocatorInterface;
+use Magento\Catalog\Ui\DataProvider\Product\Form\Modifier\AbstractModifier;
+use Magento\Ui\Component\Form\Fieldset;
+use Magento\Ui\Component\Form\Field;
+use Magento\Ui\Component\Form\Element\DataType\Text;
+
+class NewField extends AbstractModifier
+{
+    private $locator;
```

Comment:
phpdoc required here: what type of object is stored in this private field?

Reply:
Fixed.